### ISSUE TITLE
Maven repository - dbmaintain:2.7.3

### ISSUE THREAD

**Issue by lenoch7** [DbMaintain/dbmaintain#30]
Hello,
is there a plan to publish latest version of dbmaintain - 2.7.3 to Maven repository? I see only a version 2.7.2 from Dec, 2016.
 - https://search.maven.org/search?q=dbmaintain
 - https://mvnrepository.com/artifact/org.dbmaintain/dbmaintain
 - https://repo.maven.apache.org/maven2/org/dbmaintain/dbmaintain/maven-metadata.xml

Thank you.

**Comment by pSub** [comment]
Yes, we plan to publish this release. I pushed it in the upper part of my todo list and will try to get it done in this week.

**Comment by remast** [comment]
Unfortunately we tried that before but since red6 took over the project from the previous maintainers uploading to the public Maven repo was not working. There used to be problems related to the Maven group id and GPG keys. Maybe that has improved in the meantime.
